src/Lykke.Service.KycReports.Core/Settings/ServiceSettings/KycReportsSettings.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Lykke.Service.Kyc.Client;
+using Lykke.Service.PersonalData.Settings;
+
+namespace Lykke.Service.KycReports.Core.Settings
+{
+    public class KycReportsSettings
+    {
+        public DbSettings Db { get; set; }
+        public KycServiceSettings KycServiceSettings { get; set; }
+        public ServiceSettings Services { get; set; }
```

Comment:
Need to remove this settings and use external ClientAccountClient section via AccessList

Reply:
Fixed.
ClientAccountServiceClient config section is used now.